CodeContracts.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2013
-VisualStudioVersion = 12.0.31101.0
+# Visual Studio 14
```

Comment:
You are right. I was trying to leave sln file headers unchanged but failed this time. Good catch.
There are some problems with VS2015 build which I am going to address in later time.